packages/workspace/src/command-line/workspace-results.ts

```diff
@@ -38,9 +38,14 @@ export class WorkspaceResults {
           this.commandResults = commandResults;
         }
       } catch (e) {
+        /**
```

Comment:
@vsavkin @FrozenPandaz would like your thoughts on this

Reply:
we can remove the logging